### DISCUSSION TITLE
[Karate tests] allure-results is not generating up!

### DISCUSSION THREAD

**Discussion by shyam0x1e** [allure-framework/allure2#2602] [Questions & Support]
```
@Test
    void testParallel() {
        Results results = Runner.builder().hook(new AllureKarate())
                .path("classpath:tests")
                .karateEnv("dev")
                .parallel(4);
        generateReport(results.getReportDir());
        // Runner.builder().hook(new AllureKarate());
        assertTrue(results.getFailCount() == 0, results.getErrorMessages());
    }
```
 the above code has a Runner class, tests started running and completes the tests. but it is not ending. it got stuck. no such folder created `allure-results`
Can somebody please help!
    
    I am using `mvn test -Dtest=TestRunner`
<img width="995" alt="Screenshot 2024-06-16 at 4 08 40 PM" src="https://github.com/allure-framework/allure2/assets/44321046/bd60f0d5-c955-4fd0-a634-adf2767ea365">

**Comment by baev** [comment]
Could you please provide a sample repo to reproduce the issue?

**Reply by maverickwhites** [reply]
Here's a sample repo structure:

```
issue-repo/
├── README.md
├── src/
│   └── main.py
└── tests/
    └── test_main.py
```

Include details in README.md on how to reproduce the issue in `main.py` and corresponding tests in `test_main.py` to validate it.

**Comment by shyam0x1e** [comment]
@baev can you tell me how the results are getting generated. so I that I can try if there is anyother way I can integrate this?
I am using Junit Jupiter as @Test. triggering the TestRunner using mvn. mvn is not able to find the @Karate.Test like how you have shown on your example

**Reply by shyam0x1e** [reply]
My runner class syntax will look like how you mentioned here
https://github.com/allure-framework/allure2/issues/1061#issuecomment-1881198675
